.github/PULL_REQUEST_TEMPLATE/manual_maintenance.md

```diff
@@ -23,7 +23,7 @@ appear.
   https://github.com/JLLeitschuh/ktlint-gradle/pull/667)
 - [ ] Update @zxcvbn-ts/language-common to 3.0.3 is skipped due to issues in rebuilding keycloak Docker images
 - [ ] Update @zxcvbn-ts/language-en to 3.0.1 is skipped due to issues in rebuilding keycloak Docker images
-- [ ] Flyway version from 9.18.0 and above (until 9.21.1) seem to cause issues. Future versions should be tried
+- [ ] Flyway version from 9.18.0 and above (until 9.22.1) seem to cause issues. Future versions should be tried
```

Comment:
But we migrated to this version. So either remove this line or revert the version. We should definitely specify the issues we have more clearly here.

Reply:
line removed